inspirehep/modules/records/serializers/jinja.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Jinja serializer for records."""
+
+from __future__ import absolute_import, division, print_function
+
+from invenio_records_rest.serializers.json import MarshmallowMixin, PreprocessorMixin
+
+import jinja2
+import os
+import pkg_resources
+
+
+class JinjaSerializer(MarshmallowMixin, PreprocessorMixin):
+    """Jinja serializer for records."""
+
+    def __init__(self, format, **kwargs):
+        """Initialize record."""
+        self.format = format
+        super(JinjaSerializer, self).__init__(**kwargs)
+
+    def serialize(self, pid, record, links_factory=None, **kwargs):
+        """Serialize a single record and persistent identifier.
+
+        :param pid: Persistent identifier instance.
+        :param record: Record instance.
+        :param links_factory: Factory function for record links.
+        """
+        data = self.transform_record(pid, record, links_factory, **kwargs)
+
+        return self.latex_template().render(data=data, format=self.format)
+
+    def preprocess_record(self, pid, record, links_factory=None, **kwargs):
+        """Prepare a record and persistent identifier for serialization."""
+        return record
+
+    def latex_template(self):
+        latex_jinja_env = jinja2.Environment(
+            variable_start_string='\VAR{',
+            variable_end_string='}',
+            loader=jinja2.FileSystemLoader(os.path.abspath('/'))
+        )
+        template_path = pkg_resources.resource_filename('inspirehep', 'modules/records/serializers/templates/latex_template.tex')
+
+        template = latex_jinja_env.get_template(template_path)
```

Comment:
I'd suggest to move this logic in https://github.com/inspirehep/inspire-next/blob/8aefa86d3d9be0d4f1f08c852c84a2c8ff73db7c/inspirehep/modules/records/ext.py 

even better registered them in the blueprint and use them as usual:

```python
from flask import render_template

 return render_template('inspire_records/template_name', foo='bar', bar='foo')

```